saleor/checkout/tests/test_checkout.py

```diff
@@ -40,6 +27,19 @@
     recalculate_checkout_discount,
     remove_voucher_from_checkout,
 )
+from ...account import CustomerEvents
+from ...account.models import Address, CustomerEvent, User
+from ...account.utils import store_user_address
+from ...core.exceptions import InsufficientStock
+from ...core.taxes import zero_money, zero_taxed_money
+from ...discount import DiscountValueType, VoucherType
+from ...discount.models import NotApplicable, Voucher
+from ...order import OrderEvents, OrderEventsEmails
+from ...order.models import OrderEvent
+from ...payment.models import Payment
+from ...plugins.manager import get_plugins_manager
+from ...shipping.models import ShippingZone
+from ...tests.utils import flush_post_commit_hooks
```

Comment:
I think this change is not needed; also `isort` fails in CI now supposedly because of it.

Reply:
I See. 
I applied the Black and it changes this import.
Isort is reverting to earlier version.

Is there any steps which i can follow while pushing PR?